vulnfeeds/cves/versions_test.go

```diff
@@ -135,339 +138,402 @@ func TestParseCPE(t *testing.T) {
 	}
 
 	for _, tc := range tests {
-		got, err := ParseCPE(tc.inputCPEString)
-		if err != nil && tc.expectedOk {
-			t.Errorf("test %q: ParseCPE for %q unexpectedly failed: %+v", tc.description, tc.inputCPEString, err)
-		}
-		if !reflect.DeepEqual(got, tc.expectedCPEStruct) {
-			t.Errorf("test %q: ParseCPE for %q was incorrect, got: %#v, expected: %#v", tc.description, tc.inputCPEString, got, tc.expectedCPEStruct)
-		}
+		tc := tc // Capture range variable
+		t.Run(tc.description, func(t *testing.T) {
+			t.Parallel()
+			got, err := ParseCPE(tc.inputCPEString)
+			if err != nil && tc.expectedOk {
+				t.Errorf("test %q: ParseCPE for %q unexpectedly failed: %+v", tc.description, tc.inputCPEString, err)
+			}
+			if !reflect.DeepEqual(got, tc.expectedCPEStruct) {
+				t.Errorf("test %q: ParseCPE for %q was incorrect, got: %#v, expected: %#v", tc.description, tc.inputCPEString, got, tc.expectedCPEStruct)
+			}
+		})
 	}
 }
 
 func TestRepo(t *testing.T) {
 	tests := []struct {
-		description     string // human-readable description of test case
-		inputLink       string // a possible repository URL to call Repo() with
-		expectedRepoURL string // The expected  repository URL to get back from Repo()
-		expectedOk      bool   // If an error is expected
+		description       string    // human-readable description of test case
+		inputLink         string    // a possible repository URL to call Repo() with
+		expectedRepoURL   string    // The expected  repository URL to get back from Repo()
+		expectedOk        bool      // If an error is expected
+		disableExpiryDate time.Time // If test needs to be disabled due to known outage.
 	}{
 		{
-			description:     "GitHub compare URL",
-			inputLink:       "https://github.com/kovidgoyal/kitty/compare/v0.26.1...v0.26.2",
-			expectedRepoURL: "https://github.com/kovidgoyal/kitty",
-			expectedOk:      true,
+			description:       "GitHub compare URL",
+			inputLink:         "https://github.com/kovidgoyal/kitty/compare/v0.26.1...v0.26.2",
+			expectedRepoURL:   "https://github.com/kovidgoyal/kitty",
+			expectedOk:        true,
+			disableExpiryDate: NO_EXPIRY,
```

Comment:
Thinking about this, I think by default, time.Time{} is a 0 value, which will always be in the past. So there's actually no need for disableExpiryDate on any of the cases where we are not disabling.